azure/pipelines/node-npm/install-volta.yml

```diff
@@ -0,0 +1,35 @@
+# Install Volta Node version manager in your pipeline
+# https://wiki.delaware.pro/pages/viewpage.action?pageId=314778333
+
+steps:
+- bash: |
+   echo "Downloading Volta..."
+   curl https://get.volta.sh | bash -s -- --skip-setup
+
+   echo "Preparing environment variables..."
+   export VOLTA_HOME="$HOME/.volta"
+   export PATH="$VOLTA_HOME/bin:$PATH"
+   
+   echo "##vso[task.setvariable variable=VOLTA_HOME]$VOLTA_HOME"
+   echo "##vso[task.setvariable variable=PATH]$PATH"
+   
+   echo "Starting installation..."
+   volta setup --verbose
+  condition: or(eq( variables['Agent.OS'], 'Linux' ), eq( variables['Agent.OS'], 'Darwin' ))
+  displayName: Install Volta for Unix
+- powershell: |
+   $voltaVersion = "1.1.0"
+   $voltaFileName = "volta-${voltaVersion}-windows-x86_64.msi"
+   Write-Host "Downloading ${voltaFileName}..."
+   Invoke-WebRequest https://github.com/volta-cli/volta/releases/download/v${voltaVersion}/${voltaFileName} -OutFile $(System.DefaultWorkingDirectory)\${voltaFileName}
```

Comment:
$(Agent.ToolsDirectory) might be more suitable, so that the exe will not interfere with assumptions taken during build or  publish.